qrexec/utils.py

```diff
@@ -117,12 +131,12 @@ def get_system_info():
     """
 
     system_info = qubesd_call("dom0", "internal.GetSystemInfo")
-    return json.loads(system_info.decode("utf-8"))
+    return cast(SystemInfo, json.loads(system_info.decode("utf-8"))["domains"])
```

Comment:
I had a bad feeling about this change and now I know why. The policy modules are used in other places too, for example [here](https://github.com/QubesOS/qubes-core-admin/blob/main/qubes/ext/admin.py#L76). So, this API change (expected layout of `system_info`) broke them. While it's theoretically okay to change the other places too (not too many of them), there is also another concern: this `domains` was put there for a reason - to leave a place if ever policy would need some more info (like global properties). I don't think there is much performance impact with this extra level, but also I see it will make the annotations a bit more complex. So, if I haven't missed any more important reason to eliminate it, I'd prefer to leave it.